src/frontends/pytorch/src/transforms/prim_tuple_construct_replacer.cpp

```diff
@@ -29,7 +29,20 @@ bool DecomposeTupleResults::run_on_model(const std::shared_ptr<Model>& model) {
         }
         auto inputs = input_node->inputs();
         for (auto input : inputs) {
-            model->add_results({std::make_shared<opset8::Result>(input.get_source_output())});
+            auto out = input.get_source_output();
+            auto const_out = cast_fw_node(out.get_node_shared_ptr(), "prim::Constant");
```

Comment:
This doesn't guarantee that constant is NoneType, it can be string constant

Reply:
removed for now